packages/attention/src/component.tsx

```diff
@@ -91,8 +114,8 @@ export function Attention(props: AttentionProps) {
   return (
     <div
       aria-describedby={ariaDescribedby}
-      role={ariaRole}
-      aria-label={ariaLabel}
+      role={props.tooltip ? 'tooltip' : 'img'}
+      aria-label={`${activeAttentionProp()} speech bubble pointing ${pointingAt()}`}
```

Comment:
I've also just realised that the whole "pointing at ..." should not be added if `props.noArrow === false`.

Reply:
wait shouldn't it be the opposite? that if we should not add "pointing at..." of props.noArrow === true? Because if don't have an arrow we should say "pointing at..."? :)

Reply:
Of course, it was supposed to be `props.noArrow === true` 😄